crates/robot-panic/src/lib.rs

```diff
@@ -29,7 +29,7 @@
+//! You can submit an issue with the crash report at this link: https://github.com/apollographql/rover/issues/new?title=bug%3A+crashed+while+%3Cinsert+description+here%3E&body=%3C%21--%0A++Please+add+some+additional+information+about+what+you+were+trying+to+do+before+submitting+this+report%0A+--%3E+%0A%0A**Crash+Report**%0A+%60%60%60toml%0Aname+%3D+%27rover%27%0Aoperating_system+%3D+%27unix%3AUbuntu%27%0Acrate_version+%3D+%270.0.0%27%0Aexplanation+%3D+%27%27%27%0APanic+occurred+in+file+%27src%2Fbin%2Frover.rs%27+at+line+11%0A%27%27%27%0Acause+%3D+%27oh+no%27%0Amethod+%3D+%27Panic%27%0Abacktrace+%3D+%27%27%27%0A%0A+++0%3A+0x5575363b5003+-+std%3A%3Asys_common%3A%3Abacktrace%3A%3A__rust_begin_short_backtrace%3A%3Ahcc063f8de39c7379%0A+++1%3A+0x5575363b505d+-+std%3A%3Art%3A%3Alang_start%3A%3A%7B%7Bclosure%7D%7D%3A%3Ahf77d52b639388bce%0A+++2%3A+0x5575364a2e41+-+core%3A%3Aops%3A%3Afunction%3A%3Aimpls%3A%3A%3Cimpl+core%3A%3Aops%3A%3Afunction%3A%3AFnOnce%3CA%3E+for+%26F%3E%3A%3Acall_once%3A%3Ah6a3209f124be2235%0A++++++++++++++++at+%2Frustc%2F18bf6b4f01a6feaf7259ba7cdae58031af1b7b39%2Flibrary%2Fcore%2Fsrc%2Fops%2Ffunction.rs%3A259%0A+++++++++++++++++-+std%3A%3Apanicking%3A%3Atry%3A%3Ado_call%3A%3Ah88ce358792b64df0%0A++++++++++++++++at+%2Frustc%2F18bf6b4f01a6feaf7259ba7cdae58031af1b7b39%2Flibrary%2Fstd%2Fsrc%2Fpanicking.rs%3A373%0A+++++++++++++++++-+std%3A%3Apanicking%3A%3Atry%3A%3Ah6311c259678e50fc%0A++++++++++++++++at+%2Frustc%2F18bf6b4f01a6feaf7259ba7cdae58031af1b7b39%2Flibrary%2Fstd%2Fsrc%2Fpanicking.rs%3A337%0A+++++++++++++++++-+std%3A%3Apanic%3A%3Acatch_unwind%3A%3Ah56c5716807d659a1%0A++++++++++++++++at+%2Frustc%2F18bf6b4f01a6feaf7259ba7cdae58031af1b7b39%2Flibrary%2Fstd%2Fsrc%2Fpanic.rs%3A379%0A+++++++++++++++++-+std%3A%3Art%3A%3Alang_start_internal%3A%3Ah73711f37ecfcb277%0A++++++++++++++++at+%2Frustc%2F18bf6b4f01a6feaf7259ba7cdae58031af1b7b39%2Flibrary%2Fstd%2Fsrc%2Frt.rs%3A51%0A+++3%3A+0x5575363b4fc2+-+main%0A+++4%3A+0x7fa1956800b3+-+__libc_start_main%0A+++5%3A+0x5575363b40be+-+_start%0A+++6%3A++++++++0x0+-+%3Cunresolved%3E%27%27%27%0A%0A%60%60%60%0A
```

Comment:
rotten diff here but this just removes a double slash after `apollographql/rover`